tools/Dockerfile

```diff
@@ -46,16 +45,15 @@ COPY . .
 
 WORKDIR /app/flipperzero-rs/tools
 
-RUN cargo run --bin generate-bindings /app/firmware/build/f7-firmware-D/sdk_headers/
+# We don't use debian's libclang. For details see https://github.com/flipperzero-rs/flipperzero/pull/70#discussion_r1199723419
+RUN env LIBCLANG_PATH=/app/clang+llvm-16.0.4-x86_64-linux-gnu-ubuntu-22.04/lib cargo run --bin generate-bindings /app/firmware/build/f7-firmware-D/sdk_headers/
 
 RUN cp bindings.rs ../crates/sys/src
 
-RUN rustfmt --config newline_style=unix ../crates/sys/src/bindings.rs
```

Comment:
Explicitly running `rustfmt` is not necessary since bindgen runs `rustfmt` by default.
See https://rust-lang.github.io/rust-bindgen/code-formatting.html

Reply:
I think this was added because for whatever reason, the first time the `Dockerfile` was run, everything was left unformatted on a single line. If that's no longer happening, great!